docs/content/advanced/crunchy-postgres-exporter.md

```diff
@@ -0,0 +1,53 @@
+---
+title: "Crunchy Postgres Exporter"
+date: 
+draft: false
+weight: 3
+---
+
+The crunchy-postgres-exporter container provides real time metrics about the PostgreSQL database
+via an API. These metrics are scraped and stored by a [Prometheus](https://prometheus.io)
+time-series database and are then graphed and visualized through the open source data
+visualizer [Grafana](https://grafana.com/).
+
+The crunchy-postgres-exporter container uses [pgMonitor](https://github.com/CrunchyData/pgmonitor) for advanced metric collection.
+It is required that the `crunchy-postgres-ha` container has the `PGMONITOR_PASSWORD` environment
+variable to create the appropriate user (`ccp_monitoring`) to collect metrics.
+
+Custom queries to collect metrics can be specified by the user. By
+mounting a **queries.yml** file to */conf* on the container, additional metrics
+can be specified for the API to collect. For an example of a queries.yml file, see
+[here](https://github.com/CrunchyData/pgmonitor/blob/master/exporter/postgres/queries_common.yml)
+
+## Packages
+
+The crunchy-postgres-exporter Docker image contains the following packages (versions vary depending on PostgreSQL version):
+
+* PostgreSQL (12.3, 11.8, 10.13, 9.6.18 and 9.5.22)
+* CentOS7 - publicly available
+* UBI7 - customers only
+* [PostgreSQL Exporter](https://github.com/wrouesnel/postgres_exporter)
+
+## Environment Variables
```

Comment:
Should or will the user ever be directly modifying the env vars for this container themselves?

We don't typically document Operator-specific container env vars since the user influences config via `pgcluster` specs (and therefore CLI flags, etc.) and not directly via env vars, so wanted to clarify the motivation here.

Reply:
The updated documentation provides the info needed to connect the various metrics available through the container, so the extra details may be more useful here than for other containers in the project.

Since this was posted, I added some more details to hopefully provide better context. Let me know if there might be anything else that should be added or deleted, and I'm happy to tweak what's here.